Source/OxyPlot.Avalonia/Utilities/Keyboard.cs

```diff
@@ -24,7 +26,7 @@ internal class Keyboard
 
         public Keyboard()
```

Comment:
Again, not specific to this PR, but can we replace this with `ModifierKeys = e.KeyModifiers.ToModifierKeys(),` in `ToMouseWheelEventArgs` and remove this class?

Reply:
removed.